apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/shared.tsx

```diff
@@ -75,6 +75,8 @@ export const Bullet = ({ status, bulletText, text, subText, ...props }: BulletPr
           <Icons.CircledCheckSolidMedium size={24} color={colors.success.c60} />
         ) : status === Status.updateAvailable ? (
           <InfoCircle size={24} color={colors.constant.purple} />
+        ) : status === Status.failed ? (
+          <Icons.CircledCrossSolidMedium size={24} color={colors.error.c80} />
```

Comment:
here (and above) we could remove this `color={colors.error.c80}` and replace by `color="error.c80"`

Reply:
actually maybe not for `InfoCircle` (to verify), but for all icons coming from the react-ui package yes